src/controllers/util/index.ts

```diff
@@ -51,7 +51,7 @@ export class ControllerUtility {
 
   private static convertSourceInputv0Tov1(sourceEvents: unknown[]): SourceInput[] {
     return sourceEvents.map(
```

Comment:
@sanpj2292 let's do one quick local testing of both v0 & v1 source transformation flows taking one destination under each version

Reply:
Both the flows are working as expected